packages/uswds-core/src/styles/mixins/general/icon.scss

```diff
@@ -148,7 +158,13 @@
   @supports (mask: url("")) {
     background: none;
     background-color: if($color == currentColor, $color, color($color));
-    mask: $mask-props;
+    mask-image: url("#{$path}/usa-icons/#{$filename-base}.svg"),
+      linear-gradient(transparent, transparent);
+    mask-repeat: no-repeat;
+    mask-position: $position-x $position-y;
+    mask-size: $width $height;
+    mask-repeat: no-repeat;
+
```

Comment:
Stopped using shorthands for background and mask properties to allow user more control in overrides.